lib/processors/bundlers/moduleBundler.js

```diff
@@ -46,7 +97,15 @@ class LocatorResourcePool extends ResourcePool {
  * @param {module:@ui5/fs.Resource[]} parameters.resources Resources
  * @param {Object} parameters.options Options
  * @param {Object} parameters.options.bundleDefinition Module bundle definition
+ * @param {string} parameters.options.bundleDefinition.name The module bundle name
+ * @param {string[]} [parameters.options.bundleDefinition.defaultFileTypes=[".js", ".fragment.xml", ".view.xml", ".properties", ".json"]] List of default file types to be included in the bundle
+ * @param {ModuleBundleDefinitionSection[]} parameters.options.bundleDefinition.sections List of module bundle definition sections.
  * @param {Object} parameters.options.bundleOptions Module bundle options
+ * @param {boolean} [parameters.options.bundleOptions.optimize=false] If set to 'true' the module bundle gets minified
+ * @param {boolean} [parameters.options.bundleOptions.decorateBootstrapModule=true] If set to 'false', the module won't be decorated with an optimization marker
+ * @param {boolean} [parameters.options.bundleOptions.addTryCatchRestartWrapper=false] Whether to wrap bootable module bundles with a try/catch to filter out "Restart" errors
+ * @param {boolean} [parameters.options.bundleOptions.usePredefineCalls=false] If set to 'true', sap.ui.predefine is used for UI5 modules
```

Comment:
@codeworrior how does this relate to the "preload" mode? There we currently only document that `sap.ui.predefine` is used